skyportal/facility_apis/atlas.py

```diff
@@ -0,0 +1,310 @@
+import requests
+import json
+from datetime import datetime, timedelta
+from astropy.time import Time
+import numpy as np
+import pandas as pd
+from io import StringIO
+
+from . import FollowUpAPI
+from . import FakeApplication, FakeRequest
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+if cfg['app.atlas.port'] is None:
+    ATLAS_URL = f"{cfg['app.atlas.protocol']}://{cfg['app.atlas.host']}"
+else:
+    ATLAS_URL = (
+        f"{cfg['app.atlas.protocol']}://{cfg['app.atlas.host']}:{cfg['app.atlas.port']}"
+    )
+
+
+class ATLASRequest:
+
+    """A dictionary structure for ATLAS forced photometry requests."""
+
+    def _build_payload(self, request):
+        """Payload json for ATLAS forced photometry requests.
+
+        Parameters
+        ----------
+
+        request: skyportal.models.FollowupRequest
+            The request to add to the queue and the SkyPortal database.
+
+        Returns
+        ----------
+        payload: json
```

Comment:
```suggestion
        payload : dict
```

`dict`, unless you return a string specifically formatted according to JSON standards